plotter/histogram_test.go

```diff
@@ -62,3 +63,33 @@ func ExampleHistogram() {
 func TestHistogram(t *testing.T) {
 	cmpimg.CheckPlot(ExampleHistogram, t, "histogram.png")
 }
+
+func TestSingletonHistogram(t *testing.T) {
+	done := make(chan struct{}, 1)
+	go func() {
+		defer close(done)
+		p, err := plot.New()
+		if err != nil {
+			t.Errorf("unexpected error from plot.New: %v", err)
```

Comment:
nit: consider `t.Fatalf()` if the rest of the test could panic if the condition fails.